Makefile.am

```diff
@@ -37,6 +37,9 @@ distclean-local::
 		-o -name '*.gcno' \) \
 		-type f -print | xargs $(RM)
 
+all-local:
+	@scripts/zfs-tests.sh -c
```

Comment:
I was close, actually this should be `-${top_srcdir}/scripts/zfs-tests.sh -c`.  FYI the leading `-` will prevent a non-zero return code from being fatal, which seems a bit harsh in this case.